frontend/src/components/main/main_page.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+class MainPage extends React.Component {
+
+  render() {
+    return (
+      <div>
+        <h1>Field Lily</h1>
+        <footer>
+        BSD 3 &copy; 2021
```

Comment:
your question about copyright makes total sense! yeah if you want to use CC-BY 4.0 feel free.  I thought you were talking about the codebase itself.  [other opensource projects](https://www.gimp.org/) have used CC licenses for the websites proper of their projects.